services/tenant-ui/frontend/package.json

```diff
@@ -10,6 +10,8 @@
     "lint": "eslint --ext .ts,vue --ignore-path .gitignore ."
   },
   "dependencies": {
+    "@vuelidate/core": "^2.0.0-alpha.44",
```

Comment:
ALPHA!!! yikes... 😉

Reply:
Hey the guy is on vacation!
https://github.com/vuelidate/vuelidate/issues/1078#issuecomment-1171959960